src/components/Navbar.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { FaArrowLeft, FaMicrophone } from 'react-icons/fa';
+import { HiCog } from 'react-icons/hi';
+import '../styles/Navbar.css';
+
+function Navbar() {
+  return (
+    <nav>
+      <div className="header">
+        <ul className="back-stats">
+          <li><NavLink to="/"><FaArrowLeft /></NavLink></li>
+          <li>WorldWide Covid Statistics</li>
+          <li className="mic-settings">
+            <div><FaMicrophone /></div>
```

Comment:
Great job @BezzB 🤝

- [ ] As stated by the previous reviewer, kindly include some form of filtering by categories or a search.